tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field/default/exception/field_exception_default_test.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:serverpod_cli/analyzer.dart';
+import 'package:serverpod_cli/src/analyzer/code_analysis_collector.dart';
+import 'package:serverpod_cli/src/analyzer/models/stateful_analyzer.dart';
+import 'package:serverpod_cli/src/generator/code_generation_collector.dart';
+import 'package:serverpod_cli/src/test_util/builders/generator_config_builder.dart';
+import 'package:serverpod_cli/src/test_util/builders/model_source_builder.dart';
+import 'package:test/test.dart';
+
+void main() {
+  var config = GeneratorConfigBuilder().build();
+
+  group('Given an exception with fields with a "default" keyword', () {
+    test(
+      'when the field is of type String and the default is set to "Default error message", then the field should have a "default model" value',
```

Comment:
Question/Discussion: If we would remove support for default values on Models along with all the tests there, we would have no validation that it would still work for exceptions right?

In theory, we could remove support for exceptions then as well. I therefore think it might be good to duplicate the tests to ensure we always have full support.

Reply:
This also makes it clear that the shared logic is used also for serializable exceptions.